### PR TITLE
OEUI-118: Widen drug prescription field boxes

### PR DESCRIPTION
**Pull request by larrystone**
## **JIRA TICKET NAME:**
[OEUI-118: Widen drug prescription field boxes to accommodate available options.](https://issues.openmrs.org/browse/OEUI-118)

### SUMMARY:
These text and combo boxes should be widened to accommodate the options...

 - the text box at the top of the screen for the drug names should be widened.  e.g. some PIH implementations have a drug name as long as "acitracin 500 internatinal units/gram + neomycin sulphate 0.5 % (0.35 % base) skin-ointment, 15 gram tube".  It doesn't necessary need to be as long as this, but we should make it as long as we can on that page.

 - The Dose and Dispensing Units box should be long enough for "International Units"

 -  The Duration Units should be long enough for "number of occurrences"

 

There could still be issues, as the text in other languages may be longer, but we should make these at least long enough for the English values.

### CODE REVIEW

File: app/js/components/orderEntry/addForm/StandardDose.jsx [hunk 1]
```diff
@@ -234,18 +234,18 @@ const StandardDose = ({
                   name="dispensingUnit"
                   id="drugDispensingUnits"
                   list="dispensingUnits"
-                  size="8"
+                  size="20"
```
[the remark this answers is not in the document]
Comment (larrystone) [reply app/js/components/orderEntry/addForm/StandardDose.jsx]: Thanks. I have updated the PR accordingly

File: app/css/openmrs-owa-orderentry.css [hunk 2]
```diff
@@ -244,7 +244,7 @@ body {
     max-height: 500px;
     overflow-x: scroll;
     background-color: #fff;
-    width: 352px;
+    width: 622px;
```
Comment (paradisekelechi) [inline app/css/openmrs-owa-orderentry.css]: What is the behaviour of this on smaller screens? @larrystone

**Comment by dkayiwa** [comment]
Some merge conflicts here.

**Comment by larrystone** [comment]
@dkayiwa thanks, I have done that

**Comment by larrystone** [comment]
@dkayiwa merge conflicts fixed again here

**Pull request merged**